### PR TITLE
adding libfyaml

### PR DESCRIPTION
**Pull request by MarDiehl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MarDiehl** [comment]
@conda-forge-admin, please ping team

File: recipes/libfyaml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{% set build_number = 0 %}
```
Comment (ocefpaf) [inline recipes/libfyaml/meta.yaml:1]:
```suggestion
```

File: recipes/libfyaml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+{% set version = "0.7.12" %}
+{% set sha256 = "485342c6920e9fdc2addfe75e5c3e0381793f18b339ab7393c1b6edf78bf8ca8" %}
```
Comment (ocefpaf) [inline recipes/libfyaml/meta.yaml:3]:
```suggestion
```

File: recipes/libfyaml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+package:
+  name: libfyaml
+  version: {{ version }}
+
+source:
+  url: https://github.com/pantoniou/libfyaml/releases/download/v{{ version }}/libfyaml-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (ocefpaf) [inline recipes/libfyaml/meta.yaml:11]:
```suggestion
  sha256: 485342c6920e9fdc2addfe75e5c3e0381793f18b339ab7393c1b6edf78bf8ca8
```

File: recipes/libfyaml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+build:
+  skip: true  # [win]
+  script:
+    - ./configure --prefix="${PREFIX}"
+    - make
+    - make install
```
Comment (ocefpaf) [inline recipes/libfyaml/meta.yaml:18]: Is there a `make check/test`?

File: recipes/libfyaml/meta.yaml (hunk 1 above)
Comment (MarDiehl) [reply recipes/libfyaml/meta.yaml:1]: is this not considered best practice? I find it convenient to have everything to change when updating a package at the top of the recipe.

**Comment by MarDiehl** [comment]
@conda-forge-admin, please ping team

**Comment by MarDiehl** [comment]
@ocefpaf: I did a force push but the CI was not started. How can I trigger it?

File: recipes/libfyaml/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/libfyaml/meta.yaml:1]: The autoupdate bot will do most of the changes for you once this is merge and too much jinja for vatiables that are used only once make the recipe hard to read. So you can say that it is not considered the best pratice.

**Pull request merged**